src/sidebar/index.js

```diff
@@ -1,11 +1,44 @@
-import { parseJsonConfig } from '../boot/parse-json-config';
+// Load polyfill for :focus-visible pseudo-class.
+import 'focus-visible';
+// The entry point component for the app.
+import { render } from 'preact';
+// Enable debugging checks for Preact. Removed in prod builds by Rollup config.
+import 'preact/debug';
 
-import { checkEnvironment } from './config/check-env';
+import { parseJsonConfig } from '../boot/parse-json-config';
+// Utilities.
+import { Injector } from '../shared/injector';
+import HypothesisApp from './components/HypothesisApp';
+import LaunchErrorPanel from './components/LaunchErrorPanel';
 import { buildSettings } from './config/build-settings';
+import { checkEnvironment } from './config/check-env';
 import {
   startServer as startRPCServer,
   preStartServer as preStartRPCServer,
 } from './cross-origin-rpc.js';
+import { ServiceContext } from './service-context';
+import { AnnotationActivityService } from './services/annotation-activity';
+// Services.
```

Comment:
This comment is now out of place. It should be above the previous line. Given that all the service imports come from a directory named `./services` it isn't that important though. We could just remove it. Same for the "Utilities" section indicator.